### PR TITLE
Bump vsearch version and add .gz support

### PR DESCRIPTION
**Pull request by colinbrislawn**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This updates to the newest release of vsearch, also also supports reading `.gz` files directly.

### CODE REVIEW

File: recipes/vsearch/meta.yaml [hunk 1]
```diff
@@ -21,9 +21,11 @@ requirements:
     - m4
     - gcc  # [linux]
     - llvm  # [osx]
-    - perl 
+    - perl
+    - zlib
   run:
     - libgcc  # [linux]
+    - zlib
```
Comment (bgruening) [inline recipes/vsearch/meta.yaml]: can you please pin to {{CONDA_ZLIB}}*

File: recipes/vsearch/meta.yaml [hunk 2]
```diff
@@ -22,10 +22,10 @@ requirements:
     - gcc  # [linux]
     - llvm  # [osx]
     - perl
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
   run:
     - libgcc  # [linux]
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
```
Comment (colinbrislawn) [inline recipes/vsearch/meta.yaml]: @bgruening Did I pin this right? I'm just imitating other recipes...

**Comment by bgruening** [comment]
perfect! Thanks a lot!

**Comment by mdehollander** [comment]
The osx build failed, but without a clear error and it seems to be that the vm just crashed. I restarted the build to see if what happens: https://travis-ci.org/bioconda/bioconda-recipes/jobs/283911124

**Comment by mdehollander** [comment]
Now the osx build also succeeds. Seems ok to me.

**Comment by colinbrislawn** [comment]
Thank you all!

**Pull request merged**